pkg/operator/ceph/object/spec.go

```diff
@@ -34,7 +37,23 @@ import (
 )
 
 const (
-	livenessProbePath = "/swift/healthcheck"
+	livenessProbePath   = "/swift/healthcheck"
+	vaultBackendPath    = "VAULT_BACKEND_PATH"
```

Comment:
Just use `secrets.VaultBackendPathKey` instead

Reply:
I was not be to find this const in libopenstorage

Reply:
Search a bit more, it's in a the vault package... github.com/libopenstorage/secrets/vault